gcc/testsuite/rust/execute/torture/builtin_macro_include_str.rs

```diff
@@ -0,0 +1,23 @@
+// { dg-output "hello, include!\n" }
+
+macro_rules! include_str {
+  () => {{}};
+}
+
+extern "C" {
+    fn printf(fmt: *const i8, ...);
+}
+
+fn print(s: &str) {
+  printf("%s" as *const str as *const i8, s as *const str as *const i8);
+}
+
+
+fn main() -> i32 {
+  // include_str! (and include_bytes!) allow for an optional trailing comma.
+  let my_str = include_str! ("include.txt",);
```

Comment:
Could you add a test that tries giving an invalid set of arguments to the macro to make sure we don't introduce regressions down the line? Something like
```rust
let file = "file.txt";
include_str!(file); // not a string literal - invalid
include_str!(); // invalid amount of arguments
include_str!("file1.txt", "file2.txt"); // same
```

Reply:
Good call. Done.